lib/store/src/StoryStore.test.ts

```diff
@@ -133,9 +133,12 @@ describe('StoryStore', () => {
   describe('componentStoriesFromCSFFile', () => {
     it('returns all the stories in the file', async () => {
       const store = new StoryStore({ importFn, globalAnnotations, fetchStoriesList });
-      await store.initialize();
+      await store.initialize({ sync: false });
 
-      const csfFile = await store.loadCSFFileByStoryId('component-one--a');
+      const csfFile = await store.loadCSFFileByStoryId({
+        storyId: 'component-one--a',
+        sync: false,
+      });
```

Comment:
Given the method name, I feel like this should take 2 arguments: `storyId: string` and `args: { sync: boolean }`